static/gsAdmin/components/customers/customerStatsFilters.tsx

```diff
@@ -8,86 +8,16 @@ import {DateTime} from 'sentry/components/dateTime';
 import {normalizeDateTimeParams} from 'sentry/components/organizations/pageFilters/parse';
 import type {ChangeData} from 'sentry/components/timeRangeSelector';
 import {TimeRangeSelector} from 'sentry/components/timeRangeSelector';
-import {DEFAULT_RELATIVE_PERIODS} from 'sentry/constants';
+import {DATA_CATEGORY_INFO, DEFAULT_RELATIVE_PERIODS} from 'sentry/constants';
 import {space} from 'sentry/styles/space';
+import {DataCategoryExact} from 'sentry/types/core';
 import type {Organization} from 'sentry/types/organization';
 import useRouter from 'sentry/utils/useRouter';
 
 const ON_DEMAND_PERIOD_KEY = 'onDemand';
 
-// TODO(data categories): for future, use DataCategory and DATA_CATEGORY_INFO to derive this information
-// Will need to update those constants to include any categories available here that aren't already defined
-export enum DataType {
-  ERRORS = 'errors',
-  TRANSACTIONS = 'transactions',
-  INDEXED_TRANSACTIONS = 'indexed_transactions',
-  REPLAYS = 'replays',
-  FEEDBACK = 'feedback',
-  PROFILES = 'profiles',
-  INDEXED_PROFILES = 'indexed_profiles',
-  MONITORS = 'monitors',
-  SPANS = 'spans',
-  INDEXED_SPANS = 'indexed_spans',
```

Comment:
the indexed data categories here are a great example of inconsistent naming; there isn't actually a need for them to be formatted as `indexed_<category>` instead of `<category>_indexed` like they are everywhere else and i have no idea why this was coded like this originally (_admin stats backend uses the snake case form of `.apiName` from `DATA_CATEGORY_INFO` anyway)